commit 5e8dc1cbf7bf18306221ec3fea57be0683107b7a
Author: Vladimir A. Nazarenko <naszar@ya.ru>
Date:   Thu Oct 9 15:37:17 2014 +1100

    staging: vt6655: delete non-standard ioctl 0xFF
    
    Custom ioctl 0xff is legacy code that was used for
    patched version of wpa_supplicant,  where function
    wpa_authen_Status_transfer() called this ioctl  to
    tell authentication   status to  the driver. While
    from upstream  was added only driver code  to  the
    kernel  but  not  patches to the wpa_supplicant we
    can safely remove this unused ioctl.
    
    Signed-off-by: Vladimir A. Nazarenko <naszar@ya.ru>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/vt6655/ioctl.c b/drivers/staging/vt6655/ioctl.c
index 970e80d92fb9..f8e990fcc6fd 100644
--- a/drivers/staging/vt6655/ioctl.c
+++ b/drivers/staging/vt6655/ioctl.c
@@ -618,38 +618,6 @@ int private_ioctl(struct vnt_private *pDevice, struct ifreq *rq)
 		pReq->wResult = 0;
 		break;
 
-#ifdef WPA_SM_Transtatus
-	case 0xFF:
-		memset(wpa_Result.ifname, 0, sizeof(wpa_Result.ifname));
-		wpa_Result.proto = 0;
-		wpa_Result.key_mgmt = 0;
-		wpa_Result.eap_type = 0;
-		wpa_Result.authenticated = false;
-		pDevice->fWPA_Authened = false;
-		if (copy_from_user(&wpa_Result, pReq->data, sizeof(wpa_Result))) {
-			result = -EFAULT;
-			break;
-		}
-
-		if (wpa_Result.authenticated == true) {
-#ifdef SndEvt_ToAPI
-			{
-				union iwreq_data wrqu;
-
-				pItemSSID = (PWLAN_IE_SSID)pMgmt->abyCurrSSID;
-
-				memset(&wrqu, 0, sizeof(wrqu));
-				wrqu.data.flags = RT_WPACONNECTED_EVENT_FLAG;
-				wrqu.data.length = pItemSSID->len;
-				wireless_send_event(pDevice->dev, IWEVCUSTOM, &wrqu, pItemSSID->abySSID);
-			}
-#endif
-			pDevice->fWPA_Authened = true; /* is successful peer to wpa_Result.authenticated? */
-		}
-		pReq->wResult = 0;
-		break;
-#endif
-
 	default:
 		pr_debug("Private command not support..\n");
 	}